tutor/templates/apps/openedx/config/cms.env.yml

```diff
@@ -9,6 +9,7 @@ FEATURES:
   {{ patch("cms-env-features")|indent(2) }}
   CERTIFICATES_HTML_VIEW: true
   PREVIEW_LMS_BASE: "{{ PREVIEW_LMS_HOST }}"
+  ENABLE_CORS_HEADERS: true
```

Comment:
Could we take this opportunity to remove `ENABLE_CORS_FEATURES` from both lms/cms.env.yml and move it to templates/apps/openedx/settings/partials/common_all.py?

    FEATURES["ENABLE_CORS_HEADERS"] = True

I'd like to have as few things as possible in the yml file, for multiple reasons, but in particular because they are not used consistently between dev and prod environment.

Reply:
Makes sense to me.  Updated.

The only thing, as you'll note, is that we have to duplicate [this section](https://github.com/openedx/edx-platform/blob/master/cms/envs/production.py#L586) because it's executed before the feature flag is actually set to true.